infra/gcb/build_project.py

```diff
@@ -195,142 +203,151 @@ def get_build_steps(project_dir):
 
   for fuzzing_engine in project_yaml['fuzzing_engines']:
     for sanitizer in get_sanitizers(project_yaml):
-      if not is_supported_configuration(fuzzing_engine, sanitizer):
-        continue
-
-      env = CONFIGURATIONS['engine-' + fuzzing_engine][:]
-      env.extend(CONFIGURATIONS['sanitizer-' + sanitizer])
-      out = '/workspace/out/' + sanitizer
-      stamped_name = name + '-' + sanitizer + '-' + ts
-      zip_file = stamped_name + '.zip'
-      stamped_srcmap_file = stamped_name + '.srcmap.json'
-      bucket = ENGINE_INFO[fuzzing_engine].upload_bucket
-      upload_url = get_signed_url(UPLOAD_URL_FORMAT.format(bucket, name, 
-                                                           zip_file))
-      srcmap_url = get_signed_url(UPLOAD_URL_FORMAT.format(bucket, name,
-                                                           stamped_srcmap_file))
-      targets_list_filename = get_targets_list_filename(sanitizer)
-      targets_list_url = get_signed_url(
-          get_targets_list_url(bucket, name, sanitizer))
-
-      env.append('OUT=' + out)
-      env.append('MSAN_LIBS_PATH=/workspace/msan')
-
-      workdir = workdir_from_dockerfile(dockerfile_path)
-      if not workdir:
-        workdir = '/src'
-
-      build_steps.append(
-          # compile
-          {
-              'name':
-                  image,
-              'env':
-                  env,
-              'args': [
-                  'bash',
-                  '-c',
-                  # Remove /out to break loudly when a build script incorrectly uses
-                  # /out instead of $OUT.
-                  # `cd /src && cd {workdir}` (where {workdir} is parsed from the
-                  # Dockerfile). Container Builder overrides our workdir so we need to add
-                  # this step to set it back.
-                  'rm -r /out && cd /src && cd {1} && mkdir -p {0} && compile'
-                  .format(out, workdir),
-              ],
-          })
+      for architecture in project_yaml['architectures']:
+        if not is_supported_configuration(fuzzing_engine, sanitizer,
+                                          architecture):
+          continue
+
+        env = CONFIGURATIONS['engine-' + fuzzing_engine][:]
+        env.extend(CONFIGURATIONS['sanitizer-' + sanitizer])
+        out = '/workspace/out/' + sanitizer
+        if architecture == 'x86_64':
+          stamped_name = '-'.join([name, sanitizer, ts])
+        else:
+          stamped_name = '-'.join([name, sanitizer, architecture, ts])
```

Comment:
might be a better idea to introduce a "directory" to separate these so that we can more easily filter out things while doing gsutil ls.

i.e. store these in gs://bucket/x86/<same format>.zip

Might even be worth considering a separate bucket.